docs/site/src/demos/lists/FolderList.js

```diff
@@ -0,0 +1,42 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import {
+  List,
+  ListItem,
+  ListItemText,
+} from 'material-ui/List';
+import Avatar from 'material-ui/Avatar';
+
+const styleSheet = createStyleSheet('FolderList', (theme) => ({
+  root: {
+    width: '100%',
+    maxWidth: '360px',
+    background: theme.palette.background.paper,
+  },
+}));
+
+const FolderList = (props, context) => {
```

Comment:
For some reason, we never use an anonymous arrow function to write our stateless functional component.
I would rather stick to this convention `function FolderList(props, context)` and use a code mod if we want to change that.